lib/common/constants.js

```diff
@@ -26,7 +26,7 @@ function constantsFactory (path) {
               error: 'red',
               warning: 'yellow',
               info: 'green',
-              debug: 'blue',
+              debug: 'cyan',
```

Comment:
@benbp totally stupid question - can you visually discern the differences here, or does this make it appear the same as the other colors?